python/cudf/dataframe/multiindex.py

```diff
@@ -465,5 +465,29 @@ def from_pandas(cls, multiindex):
                      names=multiindex.names)
         return mi
 
+    @property
+    def is_unique(self):
+        return self.codes._size == self.codes.drop_duplicates()._size
+
+    @property
+    def is_monotonic_increasing(self):
+        for i in range(1, self.codes._size):
+            if [c for c in self.codes.iloc[i]] < [
+                c for c in self.codes.iloc[i - 1]
+            ]:
+                return False
+        else:
+            return True
```

Comment:
You should be able to use the same logic as above and just loop through the columns and then make sure everything returns True as expected.

Reply:
I used pandas as a reference for the current logic.  I am not seeing a simple way to loop through columns (and apply per-column `is_monotonic` logic).  Please do let me know if I am missing something obvious (very possible), or if there is a better solution than `np.lexsort`